### PR TITLE
Feature/77  storybook

### PR DESCRIPTION
**Pull request by kevinmpowell**
Added:
* [Storybook Docs](https://github.com/EightShapes/esds-library/blob/feature/77--storybook/test/storybook/README.md)

Revised:
* [Component Dev Environment Docs](https://github.com/EightShapes/esds-library/tree/feature/77--storybook/components)

Percy VRT build is currently passing. All changes approved: https://percy.io/EightShapes-LLC/esds-storybook/builds/4375203

Closes https://github.com/EightShapes/esds-library/issues/77
Closes https://github.com/EightShapes/esds-project/issues/50
Closes https://github.com/EightShapes/esds-project/issues/52

### CODE REVIEW

File: test/storybook/README.md [hunk 1]
```diff
@@ -6,3 +6,109 @@ From this directory:
...
+![Percy](./documentation/images/percy.png)
+
+## Accessibility Testing
+The [Storybook Accessibility Addon](https://github.com/storybookjs/storybook/tree/master/addons/a11y) is enabled for all stories in the project. If the accessibility pane is not visible, choose "Show Addons" from the Storybook menu.
+
+![a11y pane](./documentation/images/a11y.gif)
+
+The accessibility addon **will not** fix all accessibility issues. An additional, manual, accessibility review step is recommended.
+
+## Debugging VRT
```
Comment (jamesmelzer) [inline test/storybook/README.md]: Move this above Accessibility Testing, to put it with the rest of the VRT sections?

**Comment by jamesmelzer** [comment]
Percy is passing, but looking at the images, it's failing to find icon, so it's VRTing an error message.

**Comment by kevinmpowell** [comment]
> Percy is passing, but looking at the images, it's failing to find icon, so it's VRTing an error message.

Yes, it's a little odd, Percy shows the diff by default, and when the _previous_ image was an error message, it makes it difficult to see what's happening. If you click on the right-hand image it will toggle the diff highlighting off so you can see the current state:

percy

File: test/storybook/README.md (hunk 1 above)
Comment (kevinmpowell) [reply test/storybook/README.md]: I debated that and thought "Hopefully this is less referenced content" and stuck it at the bottom. I trust your IA instincts though, I'll move it back.

File: test/storybook/README.md [hunk 2]
```diff
@@ -69,18 +69,18 @@ Each `<Story>` will be sent to [Percy](https://percy.io) as an individual screen
 
 ![Percy](./documentation/images/percy.png)
 
+## Debugging VRT
+`npm run build-storybook`
+
+This will create a compiled version of storybook at `./storybook-static`. This build artifact is used by VRT. If there are discrepancies between VRT results and what's shown when running the local environment, manually build this static asset and confirm it matches what VRT is displaying.
```
Comment (nathanacurtis) [inline test/storybook/README.md]: The latter sentence here is clear conceptually, but could use more concrete steps, commands, compare file(s) XYZ with file(s) ABC, etc. "Confirm it matches" is vague.

Comment (kevinmpowell) [reply test/storybook/README.md]: @nathanacurtis I've revised: https://github.com/EightShapes/esds-library/blob/feature/77--storybook/test/storybook/README.md

**Comment by kevinmpowell** [comment]
@nathanacurtis were you able to get this running locally? Any further feedback on this?

**Comment by nathanacurtis** [comment]
@kevinmpowell I have not reviewed this PR further. If you'd like me to prior to you closing, let me know.

**Pull request merged**